src/components/Composer/index.tsx

```diff
@@ -10,6 +9,8 @@ import type {AnimatedProps} from 'react-native-reanimated';
 import type {AnimatedTextInputRef} from '@components/RNTextInput';
 import RNTextInput from '@components/RNTextInput';
 import Text from '@components/Text';
+import useHtmlPaste from '@hooks/useHtmlPaste';
+import focusInputOnPaste from '@hooks/useHtmlPaste/focusInputOnPaste';
```

Comment:
nab, but I think this is more of a generic utility. And not something you'd look for inside the `@hooks` folder as it is not a hook itself.